ui_framework/components/local_nav/_local_search.scss

```diff
@@ -28,6 +28,10 @@
   }
 }
 
+.kuiLocalLuceneSearchInput:focus {
+  padding-right: 13em;
+}
```

Comment:
`ui/public/styles/local_search.less` sounds fine to me. Just to be clear, which name do you prefer in that file, `localLuceneSearchInput` or `localSearchInput-lucene`? Or something else?